src/utils/module.c

```diff
@@ -250,3 +251,66 @@ gboolean bd_utils_unload_kernel_module (const gchar *module_name, GError **error
     freelocale (c_locale);
     return TRUE;
 }
+
+/**
+ * bd_utils_get_linux_version:
+ * @error: (out): place to store error (if any)
+ *
+ * Retrieves version of currently running Linux kernel.
+ *
+ * Returns: detected Linux kernel version in the "MAJOR*1000000 + MINOR*1000 + MICRO" format.
+ */
+guint64 bd_utils_get_linux_version (GError **error)
+{
```

Comment:
Also please fix the `{}` brackets positions, this is not UDisks :-)

Reply:
Fixed (hopefully).